npm/esbuild-netbsd-64/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "esbuild-netbsd-64",
+  "version": "0.13.0",
+  "description": "The NetBSD amd64 binary for esbuild, a JavaScript bundler.",
+  "repository": "https://github.com/evanw/esbuild",
+  "license": "MIT",
+  "os": [
+    "netbsd"
```

Comment:
I'm not sure this will work because `netbsd` is not currently listed as a platform that node can return. From https://nodejs.org/api/process.html#process_process_platform:

> The `process.platform` property returns a string identifying the operating system platform on which the Node.js process is running.
>
> Currently possible values are:
>
> * `'aix'`
> * `'darwin'`
> * `'freebsd'`
> * `'linux'`
> * `'openbsd'`
> * `'sunos'`
> * `'win32'`
>
> The value `'android'` may also be returned if the Node.js is built on the Android operating system. However, Android support in Node.js [is experimental](https://github.com/nodejs/node/blob/HEAD/BUILDING.md#androidandroid-based-devices-eg-firefox-os).

This matters because esbuild's API uses `process.platform` to determine which package to take the binary executable from:

https://github.com/evanw/esbuild/blob/b8068370b3408679d53627b5e2aae7309c50239a/lib/npm/node-platform.ts#L35

This key is used to look up into the map of known packages here:

https://github.com/evanw/esbuild/blob/b8068370b3408679d53627b5e2aae7309c50239a/lib/npm/node-platform.ts#L10-L30

If `process.platform` can't return `netbsd` then running esbuild's JavaScript API on NetBSD won't work. You are welcome to [build esbuild from source](https://esbuild.github.io/getting-started/#build-from-source) from NetBSD because Go supports NetBSD, but I don't think it makes sense to publish a NetBSD binary executable to npm if node doesn't support NetBSD.